oecophylla/taxonomy/taxonomy.sh

```diff
@@ -7,35 +7,37 @@ set -e
 
 conda env create --name oecophylla-kraken -f oecophylla-kraken.yaml --quiet > /dev/null
 
+# comment out all old shogun installation scripts
+
 # currently shogun is a hack, running the install script until we
 # have stable conda install
-conda env create --name oecophylla-shogun -f oecophylla-shogun.yaml --quiet > /dev/null
+# conda env create --name oecophylla-shogun -f oecophylla-shogun.yaml --quiet > /dev/null
```

Comment:
should all these commented out lines be deleted?

Reply:
These are the instructions for older Shogun. I am testing the newer Shogun 1.0.0. There are errors... Plus the developers said that they are going to write some documentation about automated installation soon. (Current it can't be installed automatically.) I think we'd better leave these comments until we figure out a solution for the new Shogun.